### ISSUE TITLE
filetype: sass - behavior as css/scss

### ISSUE THREAD

**Issue by LArVs** [mattn/emmet-vim#211]
environment:
vim 7.4; emmet-vim updated: 2014-06-18

i'm try:

```
new test.sass
```

insert mode:

```
.test
h<C-Y>
```

result:

```
.test
  h {
    <CURSOR>
  }
```

expect:

```
.test
  height: <CURSOR>
```

**issue: by filetype: sass, not need expanding elements to brackets {} as css/scss.**

**Comment by xhoskin** [comment]
Now <Ctrl+y,n> works only if you jumping for property in brackets or quotes
But it would not jump if there is only ";" at the end of property
```
background: url(|); // would jump
font-family: "|";  // would jump
padding: ;   //would NOT jump

**Comment by sheoak** [comment]
Indeed, I was able to reproduce this. You can see *"call emmet#moveNextPrev(0)"* in the statusline. I tried with "@f" shortcut and CTRL-y + n to jump.
